tasks/main.yml

```diff
@@ -2,23 +2,33 @@
 - name: Include OS-specific variables.
   include_vars: "{{ ansible_os_family }}.yml"
 
-- name: Set the correct timezone.
+- name: Configure timezone (For Ansible 2.2+)
+  timezone:
+    name: "{{ ntp_timezone }}"
+  when: ansible_version.full >= "02.02.00.00"
```

Comment:
(I'm fine with bumping the min version since users who have to run older Ansible releases can pin to an older version of this role, and I'll bump the major version of this role when I tag a new release).